frontend/apps/DatasetDetail.vue

```diff
@@ -58,6 +58,10 @@ const emit = defineEmits([
     'delete:surface'
 ])
 
+const attachmentCount = ref(1);  // default count of the attachments 
```

Comment:
Would prefer to default to null (i.e. unknown) and do something as long as things are unknown like showing a spinner in the tab selector.